src/printer/tvmscript_printer.cc

```diff
@@ -235,16 +235,13 @@ class TVMScriptPrinter : public StmtFunctor<Doc(const Stmt&)>,
     Doc doc;
     std::ostringstream os;
     if (dtype.is_float() || dtype.is_float16() || dtype.is_bfloat16()) {
-      os.setf(std::ios::showpoint);
       os.precision(17);
     }
     os << data[0];
     if (dtype == DataType::Int(32)) {
       doc << Doc::Text(os.str());
     } else if (dtype == DataType::Bool()) {
       doc << Doc::Text(data[0] ? "True" : "False");
-    } else if (dtype == DataType::Float(32)) {
-      doc << Doc::Text(os.str());
```

Comment:
Just wanna ask why we remove these two lines?

Reply:
We want to print the float always wrapped by float32, because normally float means double.